### PR TITLE
update install bootstrap feature content urls

### PR DESCRIPTION
**Pull request by Ldcabansay**
Update question prompt with new s3 bucket url

### CODE REVIEW

File: install_bootstrap [hunk 1]
```diff
@@ -250,7 +250,7 @@ while [[ "${run_dockstore_launcher^^}" != 'Y' &&  "${run_dockstore_launcher^^}"
         ask_question "What is your Discourse category id (10 for staging, 11 for production)?" "$DISCOURSE_CATEGORY_ID" "discourse_category_id" $discourse_category_id
 
         featured_content_url='featured_content_url'
-        ask_question "What is your Featured Content URL (ex. https://s3.amazonaws.com/dockstore.featured.content.staging/develop/feat-content.html for staging, https://s3.amazonaws.com/dockstore.featured.content.staging/production/feat-content.html for production)?" "$FEATURED_CONTENT_URL" "featured_content_url" $featured_content_url
+        ask_question "What is your Featured Content URL (ex. https://s3.amazonaws.com/dockstore.featured.content/develop/feat-content.html for staging, https://s3.amazonaws.com/dockstore.featured.content/production/feat-content.html for production)?" "$FEATURED_CONTENT_URL" "featured_content_url" $featured_content_url
```
Comment (coverbeck) [inline install_bootstrap]: Small nit: in the url, you're using a branch name for staging, but a deploy env for production:

develop=staging
production=production

But give that you've set up CI and the buckets and what not, it's probably not worth changing, assuming you agree it needs to be changed.

**Comment by coverbeck** [comment]
@Ldcabansay Is this PR still open on purpose?

**Comment by denis-yuen** [comment]
@Ldcabansay  bumping, was reminded today

**Comment by Ldcabansay** [comment]
Yeah I forgot what happened with this tbh, it looks correct though. I think we have it set correctly configuration wise but I just never updated this prompt.

**Pull request merged**